packages/syft/proto/core/node/common/action/smpc_action_message.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto3";
```

Comment:
Note that there's also a packages/syft/proto/core/node/common/action/smpc_action.proto file which appears to be empty. Let's remove it.

Reply:
Yep. I need to remove them

Reply:
Is this resolved?